tests/unit/multi-year-gallery.test.js

```
@@ -0,0 +1,731 @@
+        global.HTMLElement = dom.window.HTMLElement;
+        global.CustomEvent = dom.window.CustomEvent;
+        global.Event = dom.window.Event;
+        global.KeyboardEvent = dom.window.KeyboardEvent;
+        global.requestIdleCallback = jest.fn(callback => setTimeout(callback, 0));
+        
+        // Store original values
+        originalWindow = global.window;
+        originalFetch = global.fetch;
+        
+        // Execute the source code to make MultiYearGalleryManager available
+        eval(multiYearGallerySource);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)